gulp/tasks/scripts.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+var gulp = require('gulp');
+var runSequence = require('run-sequence');
+var filenames = require('gulp-filenames');
+var concat = require('gulp-concat');
+var closureCompiler = require('gulp-closure-compiler');
+var uglify = require('gulp-uglify');
+var config = require('../config');
+
+gulp.task('compile_js', function() {
+  var closureOpts = {
+    compilerPath: './bower_components/closure-compiler/compiler.jar',
+    compilerFlags: {
+        angular_pass: true,
+        closure_entry_point: 'scaffold',
+        compilation_level: 'SIMPLE_OPTIMIZATIONS',
+        generate_exports: true,
+        manage_closure_dependencies: true,
+        only_closure_dependencies: true,
+        output_wrapper: '(function(){%output%})();',
+        js: [
+            './bower_components/closure-library/closure/**.js',
+            './bower_components/closure-library/third_party/**.js',
+            '!**_test.js'
+        ]
+    },
+    maxBuffer: 800000, // Set maxBuffer to .8GB 
+    fileName: 'build.min.js'
+  };
+
+  var externs = [];
+  externs.concat(filenames.get('closure_externs'));
```

Comment:
I see, so the `closure_externs` argument to `filenames.get` is referring to some identifier `closure_externs` set in the task and NOT the task itself, right? If that's the case, can you please rename that identifier to something else, e.g. `closure_externs_paths` to avoid confusion with the task named the same thing?

(... Or is my understanding of this not correct?)